hyperspy/io_plugins/bcf.py

```diff
@@ -1103,3 +1115,10 @@ def parse_line(line_string):
     if len(line_string) == 1:
         line_string = line_string + 'a'
     return line_string.capitalize()
+
+
+def _get_mode(obj_bcf):
+    if obj_bcf.header.sem.hv > 30.0:  # workaround to know if TEM or SEM
```

Comment:
Yes... will an optional argument. Would be an explanation in the documentation enough, instead of the logger?